### PR TITLE
Add AboutCard component to EntityPageOverview

### LINKED ISSUE #2039: Improve Information card on Overview pages
## Feature Suggestion

The information card on the overview page should show the most relevant metadata about the entity. Today it is just a simple, unweighted, list of attributes:

Screenshot 2020-08-20 at 10 34 12

Prior art (from Spotify's internal dataset information card):

Screenshot 2020-08-20 at 10 51 12

### PR DESCRIPTION
**Pull request by nikek**
## Hey, I just made a Pull Request!

Replacing the EntityMetadataCard with new design.

Screenshot 2020-08-31 at 22 59 15

fix #2039 

#### :heavy_check_mark: Checklist

- [ ] All tests are passing `yarn test`
- [ ] Screenshots attached (for UI changes)
- [ ] Relevant documentation updated
- [ ] Prettier run on changed files
- [ ] Tests added for new functionality
- [ ] Regression tests added for bug fixes

### CODE REVIEW

File: plugins/catalog/src/components/AboutCard/AboutCard.tsx [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+const useStyles = makeStyles(theme => ({
+  links: {
+    margin: theme.spacing(2, 0, 2),
+    display: 'grid',
+    gridAutoFlow: 'column',
+    gridAutoColumns: 'min-content',
+    gridGap: theme.spacing(2),
+  },
+  label: {
+    color: '#9e9e9e',
```
Comment (stefanalund) [inline plugins/catalog/src/components/AboutCard/AboutCard.tsx:45]: Will this work in dark mode?

**Comment by Fox32** [comment]
If we display the tags here in gray, I would probably switch the tags on the catalog list to gray, too.
image

They feel a bit like screaming at one 😆 I can send a PR if you are interested.

**Comment by stefanalund** [comment]
@Fox32 yeaaaah, please do! I've been meaning to bring that up as well 😄

**Comment by Fox32** [comment]
Here: #2205

File: plugins/catalog/src/components/AboutCard/AboutCard.tsx (hunk 1 above)
Comment (nikek) [reply plugins/catalog/src/components/AboutCard/AboutCard.tsx:45]: I'll see if I can find a semantic color for the label 👍

Comment (nikek) [reply plugins/catalog/src/components/AboutCard/AboutCard.tsx]: Now using `theme.palette.text.secondary` :)

**Review by marcuseide — APPROVED** [review]

**Review by soapraj — APPROVED** [review]

File: plugins/catalog/src/components/AboutCard/AboutCard.tsx (hunk 1 above)
Comment (freben) [reply plugins/catalog/src/components/AboutCard/AboutCard.tsx]: Was that not the default? I think in the fix that @Fox32 made it looks like he removed the color specification completely.

**Pull request merged**